commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java

```diff
@@ -0,0 +1,14 @@
+package io.sphere.sdk.products;
+
+/**
+ * PublishScope.
+ */
+public enum PublishScope {
+
+    /* All is the scope by default to allow to publish all changes */
+    All,
```

Comment:
Our convention is to use uppercase literals for enum, which then will be converted to the platform convention. So please change all enum literals to uppercase here.